src/coffee/import.coffee

```diff
@@ -371,7 +379,7 @@ class Import
       if filtered.shouldUpdate()
         @client.products.byId(filtered.getUpdateId()).update(filtered.getUpdatePayload())
         .then (result) =>
-          @publishProduct(result.body, rowIndex)
+          @publishProduct(result.body, rowIndex, true, publish)
```

Comment:
You could save 1 unnecessary update request by pushing publish update action as last element to filtered.getUpdatePayload() actions array during update.